typings/resin-sdk.d.ts

```diff
@@ -147,6 +147,40 @@ declare namespace ResinSdk {
 		choicesLabels?: { [key: string]: string };
 	}
 
+	interface BillingInformation {
+		id: number;
+	}
+
+	interface Organization {
+		id: number;
+		created_at: string;
+		name: string;
+
+		is_paid_for_via__billing_information: NavigationResource<
+			BillingInformation
+		>;
+
+		user__is_member_of__organization: ReverseNavigationResource<
+			OrganizationMember
+		>;
+		// TODO: re-check sbvr & whitelist
```

Comment:
Reminder: we should fix this before merging

Reply:
That was added to the main Orgs PR by https://github.com/resin-io/resin-api/pull/1162
Removing the TODO.